### ISSUE TITLE
OWASP San Juan not in the Caribbean Section

### ISSUE THREAD

**Issue by cywf** [OWASP/owasp.github.io#216]
Hey there, 

I adjusted the our owasp san juan [repo](https://github.com/owasp/www-chapter-san-juan) because it was incorrectly changed by someone (outside of our chapter btw), we (currently) are the only chapter formed in the Caribbean Islands, a fact we are quite excited to be able to potentially change / add too. So naturally, when I saw this commit reverting our chapter from the [Caribbean page](https://owasp.org/chapters/#Caribbean) to the [Central America](https://owasp.org/chapters/#CentralAmerica) section with a self approved commit from someone (again this person is not one of the leaders, or a member of the chapter, nor left any informative details as to why this change was necessary)

See the following commits for context and details:
- [Caribbean -> Central America](https://github.com/OWASP/www-chapter-san-juan/commit/3b55fe43c325596216b5490ed90287be18670943)
- [Central America -> Caribbean](https://github.com/OWASP/www-chapter-san-juan/commit/2d4339f058483e256b6ace2f6c5a9372b89a1927)
- related issue [here](https://github.com/OWASP/owasp.github.io/issues/196)
- addressing the [issue](https://github.com/OWASP/www-chapter-san-juan/commit/3b55fe43c325596216b5490ed90287be18670943#commitcomment-102161000)

TL;DR

Requesting that OWASP San Juan be bumped up to show under the Caribbean section on the chapters page [here](https://owasp.org/chapters/#Caribbean)

**Comment by securestep9** [comment]
@hblankenship this has been resolved and can be closed now
![Screenshot 2023-07-08 at 09 20 52](https://github.com/OWASP/owasp.github.io/assets/20112179/68e0beec-5de8-4a16-a018-44fd4844e5ff)

**Comment by cywf** [comment]
lgtm!
